spec.md

```diff
@@ -246,8 +249,9 @@ Policies MUST specify the following fields:
   clients SHOULD cache a policy for up to this value from last policy fetch
   time.
 * rua: Address to which aggregate feedback MAY be sent (comma-separated
-  plain-text list of email addresses, optional). For example,
-  "mailto:postmaster@example.com" from [@!RFC3986].
+  plain-text list of email addresses or an HTTPS endpoint, optional). For example,
+  "mailto:postmaster@example.com" from [@!RFC3986] or HTTP POST to "https://example.com/sts-report".
+* id: A unique identifier to identify a policy. It could be first 16 bytes of SHA256(all fields except 'id') hash
```

Comment:
In which case? I'm not sure you ever *need* to detect a policy change, but you can always diff it against the old cached policy. Since they're small and we rely on senders storing some long-lived cache of policy contents anyway, I still don't see the need for an ID. What am I missing?